test/e2e/apps/statefulset.go

```diff
@@ -1154,6 +1154,50 @@ var _ = SIGDescribe("StatefulSet", func() {
 		framework.ExpectNoError(err)
 		e2estatefulset.WaitForStatusAvailableReplicas(c, ss, 1)
 	})
+
+	ginkgo.It("AvailableReplicas should get updated accordingly when MinReadySeconds is enabled [Feature:StatefulSetMinReadySeconds] [alpha]", func() {
```

Comment:
Nit: this is beta now that we merged https://github.com/kubernetes/kubernetes/pull/104045

Reply:
thanks, fixed